docs/tutorial/t-setup.md

```diff
@@ -0,0 +1,262 @@
+## Setup Environment for the Tutorial
```

Comment:
It would be best to remove `#` from each heading (i.e. starting every page with `# Set up environment for the tutorial`, and continuing the hierarchy from there)

This can affect some important things like
- Accessibility checks
- Visual and formatting consistency among our docs
- Upcoming vanilla update (web framework that decides how our discourse content is rendered), especially automatic Table Of Content generation.